controller/dune

```diff
@@ -4,7 +4,7 @@
   (Changelog.html as Changelog.html)
   (gui/reset.css as static/reset.css)
   (gui/style.css as static/style.css)
-  (gui/vendor/focus-shift.js as static/vendor/focus-shift.js)
+  (gui/vendor/focus-shift-1-0-0.js as static/vendor/focus-shift.js)
```

Comment:
Is there an issue with using the version number in the filename, `1.0.0`?

Reply:
Yes, exactly. I think this is the conventional way, and therefore more easily recognized as a version number.

We use this same naming pattern for the ISOs here: https://github.com/dividat/playos/blob/5f6bfb694ff1e00ba824baac8f34314e79caef65/default.nix#L115-L120